src/app/projects/project-list/project-list.component.ts

```diff
@@ -61,7 +61,12 @@ export class ProjectListComponent implements OnInit {
     });
   }
 
-  onProjectRowClicked(event: MouseEvent, project: WharfProject) {
+  onProjectRowClicked(event, project: WharfProject) {
+    // Hack to avoid navigation on event bubbling from child buttons.
+    const targetClassName: string = event.target.className;
+    if (targetClassName.includes('p-button')) {
+      return;
+    }
```

Comment:
If implementing the `(click)=$event.preventDefault()` change mentioned in the comment in `project-list-item.component.html`, then this is what I mean:

```suggestion
  onProjectRowClicked(event: MouseEvent, project: WharfProject) {
    if (event.defaultPrevented) {
      return;
    }
```

Reply:
This is resolved a different path as of this comment: https://github.com/iver-wharf/wharf-web/pull/148#discussion_r871262190